packages/nextjs/app/debug/_components/contract/utilsDisplay.tsx

```diff
@@ -205,6 +210,12 @@ const _decodeContractResponseItem = (
     return decoded;
   }
 
+  // option and result are enums but we don't want to process them as enums
+  // we need the fallback so that it does not crash
+  if (isCairoOption(abiType.name || "") || isCairoResult(abiType.name || "")) {
```

Comment:
<img width="728" alt="image" src="https://github.com/user-attachments/assets/27ef54ed-8cef-4b77-9ebb-e1840512e83b">
should check `abiType.type`?

Reply:
hmm i tried to use type but it seemed to bypass this check, name works but maybe its a coincidence with the variable name (i'm using contracts from `v3-bulletproof-contracts`)

Reply:
oh i know why, here is the reason, was following this ABI, i provided a workaround to catch both `name` and `type`:

<img width="766" alt="image" src="https://github.com/user-attachments/assets/abe09edb-dc00-4950-8a5e-96eb2b27bd2c">